documentation/docs/30-advanced/67-shallow-routing.md

```diff
@@ -59,7 +59,7 @@ For this to work, you need to load the data that the `+page.svelte` expects. A c
 		href="/photos/{thumbnail.id}"
 		on:click={async (e) => {
 			// bail if opening a new tab, or we're on too small a screen
```

Comment:
It might be nice to expand this with some details about ctrl/meta on various OS to explain why we need to check both

Reply:
Does <kbd>Windows</kbd> + clicking a link open it in a new tab?

Cannot reproduce it in Windows 11 + Chrome | Firefox.

Reply:
It’s ctrl + click to open in a new tab on windows iirc

Reply:
`Windows`+click (for me, on Firefox) on a regular link does nothing - it doesn't follow it at all. It looks like it _does_ trigger a `click` event, though, so to match that behavior, we should also make sure that we bail here and let the browser use the default handler, which is to do nothing.

_However_, there are also a number of other cases we should be handling here besides meta-click. Shift-click opens in a new window, and so we want to use the default handling there. Alt-clicking (for me) does nothing, so we also want the default handling there. I'm uncertain whether we need to check that this was a left-button click.

Reply:
I believe a scroll-wheel clicking on an anchor also opens it in a new tab as well. Should this be handled as well?

Reply:
adding support for scroll-wheel clicking would be a nice touch